.github/workflows/schemas_release_pr.yml

```diff
@@ -0,0 +1,67 @@
+name: Schemas release PR
+
+on:
+  repository_dispatch:
+      types: schemas_release
+
+jobs:
+  python-dependencies:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          echo "::set-env name=PYTHON_VERSION::$(cat .python-version)"
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install apt dependencies
+        run: sudo apt-get install libsnappy-dev libgconf-2-4
+      - name: Install Pipenv
+        run: pip install pipenv==2018.11.26
+      - name: Cache virtualenv
+        id: cache-virtualenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs/
+          key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+      - name: Install virtual environment
+        if: steps.cache-virtualenv.outputs.cache-hit != 'true'
+        run: pipenv install --dev
+
+  update-schemas-version:
+    needs: python-dependencies
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          echo "::set-env name=PYTHON_VERSION::$(cat .python-version)"
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install pipenv
+        run: pip install pipenv==2018.11.26
+      - name: Cache virtualenv
+        id: cache-virtualenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs/
+          key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+      - name: Install virtual environment
+        if: steps.cache-virtualenv.outputs.cache-hit != 'true'
+        run: pipenv install --dev
+      - name: Update schemas version
+        run: pipenv run python ./scripts/update_schemas_version.py
+
+  create-pull-request:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          commit-message: Update schema release version
+          title: Automated Schema Release Updates
+          body: This is an auto-generated PR with schema release update.
+          branch: schema-release
```

Comment:
Is there a way we can get the schema version into this job? It would be good if we could make the commit message and title explicit - "Update schemas to version {version}".

Reply:
Added, thanks!👍